build.sbt

```diff
@@ -468,7 +468,7 @@ lazy val persistenceTyped = akkaModule("akka-persistence-typed")
     actorTestkitTyped % "test->test",
     clusterTyped % "test->test",
     actorTestkitTyped % "test->test",
-    jackson % "test->test")
+    jackson)
```

Comment:
Not great to add that dependency, and we would have similar problem if we would like to support classes from other modules (e.g. akka-cluster).

It's because JacksonUseAkkaSerialization is defined there. We could move that marker trait to akka-actor.

An alternative might be to have this as a StdSerializer instead of a module. Then the user would have to add an annotation to enable it. Example in docs: https://doc.akka.io/docs/akka/current/serialization-jackson.html#adt-with-trait-and-case-object
In that example the annotation is on the type (Direction), but I think it's possible to have the annotation on the field.

Reply:
The annotation would work for users but if we defined a std serializer it would still live in the jackson module and persistence would need to add it as a dep to add the annotation to the internal types we want serialized.

I'll move the marker trait.

Reply:
can be reverted now